starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java

```diff
@@ -0,0 +1,79 @@
+package com.starter.fullstack.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.starter.fullstack.api.Inventory;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@RunWith(SpringRunner.class)
+public class InventoryControllerTest {
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @Autowired
+  private MongoTemplate mongoTemplate;
+
+  @Autowired
+  private ObjectMapper objectMapper;
+
+  private Inventory inventory;
+
+  @Before
+  public void setup() throws Throwable {
+    this.inventory = new Inventory();
+    this.inventory.setId("ID");
+    this.inventory.setName("TEST");
+    // Sets the Mongo ID for us
+    this.inventory = this.mongoTemplate.save(this.inventory);
+  }
+
+  @After
+  public void teardown() {
+    this.mongoTemplate.dropCollection(Inventory.class);
+  }
+
+  /**
+   * Test findAll endpoint.
+   * @throws Throwable see MockMvc
+   */
+  @Test
+  public void findAll() throws Throwable {
+    this.mockMvc.perform(get("/inventory")
+        .accept(MediaType.APPLICATION_JSON));
+  }
+
```

Comment:
This test is causing the `GET not supported` error you were seeing. Because you removed the `@RequestMapping` this can't actually call a route, nothing matches. Because there isn't a `.andExpect(status().isOk())` here it silently fails.